util/zenfs.cc

```diff
@@ -79,41 +83,40 @@ int zenfs_tool_mkfs() {
     return 1;
   }
 
-  ZonedBlockDevice *zbd = zbd_open(false, true);
-  if (zbd == nullptr) return 1;
+  auto zbd = zbd_open(false, true);
```

Comment:
The only place I want auto to be used is for iterators in for loops - I'm very much in favor of typed languages :)